easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-system.yml

```diff
@@ -5,3 +5,7 @@ easyconfigs:
   - Nextflow-23.10.0.eb:
       options:
         from-pr: 19172
+  - CUDA-12.1.1.eb:
```

Comment:
This is not a good idea I think, since this isn't a normal CUDA installation, but a stripped down one.
I would prefer to keep track of the CUDA installations we provide in a separate, dedicated, easystack file like `eessi-2023.06-eb-4.8.2-CUDA.yml`, to avoid confusion.